src/OpalCompiler-Core/RBMethodNode.extension.st

```diff
@@ -62,6 +64,7 @@ RBMethodNode >> generate: trailer [
 { #category : #'*OpalCompiler-Core' }
 RBMethodNode >> generateIR [
 	| ir |
+	bcToASTCache := nil.
```

Comment:
This part is suspicious related to failing test

Reply:
The reset I think is good: if we re-generate the IR, the cache is (potentially) not valid.

What I want to do after this PR is to refactor to *never* cache the IR... that should simplify things a bit